packages/manager/modules/nutanix/src/dashboard/component/network-tile/template.html

```diff
@@ -44,6 +44,17 @@
         </oui-tile-description>
     </oui-tile-definition>
 
+    <!-- Modify cluster Private Bandwidth -->
+    <div data-ng-if="ctrl.loadingUpgradeOptions">
+        <oui-spinner size="s"></oui-spinner>
+    </div>
+    <oui-tile-button
+        data-ng-if="$ctrl.privateBandwidthUpgradable && !$ctrl.loadingUpgradeOptions"
+        on-click="$ctrl.goToUpgradePrivateBandwidth()"
+    >
+        <span data-translate="nutanix_cluster_modify_private_bandwidth"></span>
+    </oui-tile-button>
+
     <!-- IPFO -->
     <oui-tile-definition term="IPFO">
```

Comment:
Missing translation ?

Reply:
It's a constant.

Reply:
Then it would be better to keep it as a constant in controller or constants and bring it here. Strings in `HTML` will be difficult to track and maintain